back/config/schemas/settings.schema.json.erb

```diff
@@ -1441,6 +1441,18 @@
         }
       },
 
```

Comment:
I think I added it to all the necsarry BE files, but let me know if I missed one :)